changelog.md

```diff
@@ -4,8 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [1.10.0] - 2025-06-13
```

Comment:
Any breaking change should be a major version to properly follow semver, e.g. `2.0.0`